package.json

```diff
@@ -43,7 +43,7 @@
     "@oclif/plugin-plugins": "^1.8.3",
     "@oclif/plugin-warn-if-update-available": "^1.7.0",
     "@sendgrid/mail": "^7.2.1",
-    "@twilio/cli-core": "^5.9.1",
+    "@twilio/cli-core": "https://github.com/twilio/twilio-cli-core.git#5.26.1-rc",
```

Comment:
As per the [docs](https://docs.npmjs.com/cli/v6/commands/npm-install), If the tag does not exist in the registry data for the package, it will fail. So added tag from github directly. 
Please suggest if we have any other better way to do this.

Reply:
Makes sense to me, since it's not published to npm yet, it should be fine to go via github tags route untill the changes are pushed to `main`. 

[Q] This might not work with the `Librarian` logic for the PR you added for bumping up the version right? Or does it pick up the parsed tag version itself while bumping up?

Reply:
Currently `librarian` is picking the version from `changes.md` not from `package.json`. In librarian I have added the change to bump the version as `-rc` if the branch is not `main` but not yet pushed.

Reply:
`nit` - Might consider replacing with a shorter syntax:

```
    "@twilio/cli-core": "twilio/twilio-cli-core#5.26.1-rc",
```